web/src/components/core/Section.jsx

```diff
@@ -23,148 +23,130 @@
 
 import React from "react";
 import { Link } from "react-router-dom";
-import {
-  Button,
-  Text,
-  TextContent,
-  TextVariants,
-  Tooltip
-} from "@patternfly/react-core";
-
+import { Text, TextVariants } from "@patternfly/react-core";
 import { Icon } from '~/components/layout';
 import { ValidationErrors } from "~/components/core";
 
 /**
- * Helper method for rendering section icon
+ * Internal component for rendering the section icon
  *
- * @param {string} name
- * @param {number} [size=32]
+ * @param {object} props
+ * @param {string} [props.name] - the name of the icon
+ * @param {number} [props.size=32] - the icon size
  *
- * @return {React.ReactNode}
+ * @return {React.ReactElement}
  */
-const renderIcon = (name, size = 32) => {
+const SectionIcon = ({ name, size = 32 }) => {
   if (!name) return null;
 
   return <Icon name={name} size={size} aria-hidden />;
 };
 
+/**
+ * Internal component for rendering the section title
+ *
+ *  NOTE: a section can do either, navigate to a page following given path or
```

Comment:
np: I would drop the "NOTE: " part. It is a comment that makes perfect sense in the description of the component.

Reply:
Sure, and it is there already :)